backend/src/bundles/avatar-videos/avatar-videos.service.ts

```diff
@@ -114,6 +114,9 @@ class AvatarVideoService {
             name: getFileName(id),
             url: savedUrl,
             userId,
+            composition: JSON.stringify({}),
+            previewUrl:
+                'https://speech.microsoft.com/assets/avatar/harry/harry-business-thumbnail.png',
```

Comment:
Is this mocked data for now? Why not take the image already from the composition?

Reply:
I've already done this in my next PR.  https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/pull/318/files#diff-277a782f500c4ec4c6a53ff830a0acbb45e0d9f852afa74c0b1c2d25f54eccbeR60